run.py

```diff
@@ -20,6 +20,8 @@
 from os.path import abspath, dirname, realpath, join as joinpath
 from contextlib import contextmanager
 
+import sas.system.user
```

Comment:
I removed this import locally (it isn't used) and `run.py` successfully started SasView.

>> Running python run.py after running python setup.py clean build install, I get an error that images in the sas/qtgui/images directory cannot be found. I haven't had a chance to trace this, but I don't see any time that directory was included with the build. This error is not present when running the main branch, though, so I'm not sure why it's happening now.

>I think this is an issue with setup.py, this kind of thing is going to keep happening until its been shown some love. For now, it can probably just be added. Fixing the issues with setup.py is next on my list after this PR is accepted.